### DISCUSSION TITLE
How can I change my homepage layout ?

### DISCUSSION THREAD

**Discussion by daedalus2713** [mmistakes/minimal-mistakes#2747] [Q&A] [answered]
I can not find how to change my home page (url: <username>.gitihub.io) to about page like this https://mmistakes.github.io/minimal-mistakes/about/

Can someone show my how to config it ?

**Comment by abevoelker** [comment, marked as answer]
The source for the about page is here: https://raw.githubusercontent.com/mmistakes/minimal-mistakes/master/docs/_pages/about.md

If you create an `index.html` file and put that content in there it should render your home page similarly

**Reply by answerquest** [reply]
damn this is so different than what the quickstart site generator created
